crates/typst-library/src/text/raw.rs

```diff
@@ -149,17 +149,55 @@ pub struct RawElem {
     /// ```
     /// ````
     #[parse(
-        let (syntaxes, data) = parse_syntaxes(vm, args)?;
+        let (syntaxes, syntaxes_data) = parse_syntaxes(vm, args)?;
         syntaxes
     )]
     #[fold]
     pub syntaxes: SyntaxPaths,
 
-    /// The raw file buffers.
+    /// The raw file buffers of syntax definition files.
     #[internal]
-    #[parse(data)]
+    #[parse(syntaxes_data)]
     #[fold]
-    pub data: Vec<Bytes>,
+    pub syntaxes_data: Vec<Bytes>,
+
+    /// Load the file for syntax highlighting in the `tmTheme` file format.
+    ///
+    /// ````example
+    ///  #set raw(theme: "halcyon.tmTheme")
+    ///
+    ///  ```typ
+    ///  = Chapter 1
+    ///  #let hi = "Hello World"
+    ///  ```
```

Comment:
```suggestion
    /// #set raw(theme: "halcyon.tmTheme")
    ///
    /// ```typ
    /// = Chapter 1
    /// #let hi = "Hello World"
    /// ```
```